drivers/serial/uart_xlnx_ps.c

```diff
@@ -645,26 +1175,25 @@ static const struct uart_driver_api uart_xlnx_ps_driver_api = {
 #define UART_XLNX_PS_IRQ_CONF_FUNC(port) \
 DEVICE_DECLARE(uart_xlnx_ps_##port); \
 \
-static void uart_xlnx_ps_irq_config_0(struct device *dev) \
+static void uart_xlnx_ps_irq_config_##port(struct device *dev) \
 { \
 	IRQ_CONNECT(DT_INST_IRQN(port), \
-		    DT_INST_IRQ(port, priority), \
-		    uart_xlnx_ps_isr, DEVICE_GET(uart_xlnx_ps_##port), \
-		    0); \
+		DT_INST_IRQ(port, priority), \
+		uart_xlnx_ps_isr, DEVICE_GET(uart_xlnx_ps_##port), \
+		0); \
```

Comment:
Are these whitespace changes required? The original styling was trying to make the following lines align with `(`.

Reply:
Hi Stephanos,
no, they're not really required. I guess the first one must have looked like an unwanted side-effect of adjusting the tab spacing in VS Code to me. I've reverted those two changes.

To me, there's one more open question at this point: should the RX FIFO interrupt trigger level probably be made configurable via Kconfig? The uart_basic_api test case requires the trigger level to be 1 character, otherwise, the test will get stuck at the CR/LF check as it only ever checks the first character from the RX FIFO. For real-life operation, having an interrupt trigger at every single received character looks sorta inefficient...

Reply:
> should the RX FIFO interrupt trigger level probably be made configurable via Kconfig?

Having a config option for that wouldn't be a bad idea, but if we ever set the RX FIFO interrupt trigger level to above 1, the "receiver timeout mechanism" must be configured to ensure that the UART RX interrupt is always triggered and the RX FIFO doesn't just sit around part-filled not notifying the system.